card/spec/models/card/trash_spec.rb

```diff
@@ -36,8 +36,8 @@
 
 describe Card, "in trash" do
   it "should be retrieved by fetch with new" do
-    Card.create(:name=>"Betty").delete
-    c=Card.fetch "Betty", :new=>{}
+    Card.create(name: "Betty").delete
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.